src/Text/KirbyTags.php

```diff
@@ -33,7 +34,12 @@ public static function parse(string $text = null, array $data = [], array $optio
         return preg_replace_callback($regex, function ($match) use ($data, $options) {
             try {
                 return static::$tagClass::parse($match[0], $data, $options)->render();
+            } catch (InvalidArgumentException $e) {
+                return $match[0];
             } catch (Exception $e) {
+                if (option('debug') === true) {
```

Comment:
To increase portability, this should use the provided `$options` array:

```suggestion
                if (($options['debug'] ?? false) === true) {
```